backend/pick-git/src/main/java/com/woowacourse/pickgit/post/domain/dto/RepositoryResponseDto.java

```diff
@@ -1,17 +1,27 @@
 package com.woowacourse.pickgit.post.domain.dto;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 public class RepositoryResponseDto {
 
     private String name;
 
+    @JsonProperty("html_url")
+    private String htmlUrl;
```

Comment:
```suggestion
    private String url;
```

Github에서 반환되는 URL을 `@JsonProperty`로 맵핑시켜서, 변수명은 그냥 `url`로 하는 게 더 자연스럽지 않나 싶어요! :)

Reply:
오호 그러네요! 변경하겠습니다 🙌